.idea/eclipseCodeFormatter.xml

```diff
@@ -11,4 +11,4 @@
       </ProjectSpecificProfile>
     </option>
   </component>
-</project>
+</project>
```

Comment:
Nit: can you revert this?

Reply:
Sure, i guess this happened when i was trying to configure the eclipseCodeFormatter plugin, but i'll revert this in my next commit.